### PR TITLE
Usitc zotero patch 1

### PR DESCRIPTION
**Pull request by USITC-Zotero**
Redrafted the U.S. International Trade Commission's style. The most significant fix was a conditional to route bibliography citations to treat features of certain item types  different (with periods rather than commas). Other fixes are a bit more minor and are included in the commit message.

### CODE REVIEW

**Comment by USITC-Zotero** [comment]
I am pulling this request while I resolve the conflicts then resubmit.

**Review by USITC-Zotero — COMMENTED** [review]
resolved validation test failures

**Review by USITC-Zotero — COMMENTED** [review]
We removed the point-locator-join-with-comma macro. Now, let's see if it passes the tests

**Comment by adam3smith** [comment]
Thanks -- looks like you didn't start from a checkout that included your last round of changes, so there are a bunch of merge conflicts. Do you see the "Resolve conflicts" button above (not sure for whom that's displayed)? If so, could you please handle these conflicts -- there are too many for me to handle reasonably quickly.

**Review by USITC-Zotero — COMMENTED** [review]
Got an unexpected test failure after resolving. I opted to re-try the code once more. If this fails, I'll close the pull request and open a new one.

**Comment by USITC-Zotero** [comment]
@adam3smith I'm sorry this pull request is such a headache. I've resolved the conflicts and test failure. Feel free to let us know if any additional fixes or changes are needed. Thank you for taking the time to review these repos

File: united-states-international-trade-commission.csl [hunk 1]
```diff
@@ -154,11 +154,18 @@
           </choose>
         </else-if>
         <else-if type="broadcast motion_picture" match="any">
-          <names variable="director"/>
+          <names variable="director">
+            <name form="short" delimiter="" and="text" sort-separator=" "/>
```
Comment (adam3smith) [inline united-states-international-trade-commission.csl:158]: What's the idea of `delimiter=""` here? That seems wrong

**Comment by adam3smith** [comment]
No worries -- the hassle was mainly for you. I have one quick question on the code above, rest looks good.

File: united-states-international-trade-commission.csl (hunk 1 above)
Comment (USITC-Zotero) [reply united-states-international-trade-commission.csl]: Good catch. The issue seems isolated to broadcasts/videos with 3 directors (which should be rare).  I'll change the code to `delimiter=", "`

Also, I need to make an adjustment to the contributor macro to fix name formatting in bibliographies.

**Review by USITC-Zotero — COMMENTED** [review]
We adjusted contributors and contributor-note macros.

**Review by USITC-Zotero — COMMENTED** [review]
Code is updated and ready for review. We fixed the contributor and contributor-note macros.

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**